interfaces/apparmor/template.go

```diff
@@ -635,6 +638,9 @@ profile snap-update-ns.###SNAP_INSTANCE_NAME### (attach_disconnected) {
   # Allow reading /proc/version. For release.go WSL detection.
   @{PROC}/version r,
 
+  # Allow reading somaxconn, required in newer distro releases
+  @{PROC}/sys/net/core/somaxconn r,
```

Comment:
AFAIK Go runtime is notorious for this and triggers SELinux denials.

Reply:
Yeah...